examples/chip-tool/BUILD.gn

```diff
@@ -22,6 +22,7 @@ assert(chip_build_tools)
 declare_args() {
   # Use a separate eventloop for CHIP tasks
   config_use_separate_eventloop = true
+  chip_enable_trace_chip = false
```

Comment:
Please add docs which should be a comment on the preceding line.

Reply:
Removed